pkg/operator/k8sutil/deployment.go

```diff
@@ -144,6 +144,23 @@ func WaitForDeploymentToStart(ctx context.Context, clusterdContext *clusterd.Con
 	return fmt.Errorf("gave up waiting for deployment %q to update", deployment.Name)
 }
 
+func WaitForPodsToRun(ctx context.Context, clientset kubernetes.Interface, namespace, label string) error {
+	// wait for the pods to run for a maximum of 100 sec
+	sleepTime := 5
+	attempts := 20
+	for i := 0; i < attempts; i++ {
+		allRunning, err := PodsWithLabelAreAllRunning(ctx, clientset, namespace, label)
+		if err != nil {
+			errors.Wrapf(err, "While waiting for all pods with label %s to be in running state", label)
+		}
+		if !allRunning {
+			logger.Infof("Still waiting for all pods with label %s to be in running state", label)
```

Comment:
We need to return when they are all running. What about this?
```suggestion
	if allRunning {
		logger.Info("All mgr pods are running")
		return nil
	}
	logger.Infof("Still waiting for all pods with label %s to be in running state", label)
```

Reply:
Fixed